Makefile

```diff
@@ -1,3 +1,10 @@
+init:
+	wehich brew || /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
Did you mean `which`?

```suggestion
	which brew || /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Reply:
I'll save up on a review cycle given the time difference and assume the answer is yes. :) Committed

Reply:
heh thanks :)